tokio/src/io/util/mem.rs

```diff
@@ -241,3 +243,17 @@ impl AsyncWrite for Pipe {
         Poll::Ready(Ok(()))
     }
 }
+
+cfg_coop! {
+    fn poll_proceed_and_make_progress(cx: &mut task::Context<'_>) -> Poll<()> {
+        let coop = ready!(crate::coop::poll_proceed(cx));
+        coop.made_progress();
+        Poll::Ready(())
+    }
+}
```

Comment:
We aren't doing this conditional thing in `src/io/util/empty.rs`. Why is it needed here?

Reply:
I'm not sure what you mean. It looks like it's in there: https://github.com/tokio-rs/tokio/blob/d6143c956603318baa640a83c8275b5891006766/tokio/src/io/util/empty.rs#L78

Reply:
Hi! Thanks for the review!

Maybe I read it wrong but I thought this is added in https://github.com/tokio-rs/tokio/blob/master/tokio/src/io/util/empty.rs#L78-L90 so I thought we only want this under coop feature. I can change it real quick if this is not the plan.

Reply:
Oh, seems like I'm just blind.